ci/config.yaml

```diff
@@ -6,8 +6,8 @@ os:
   #  - "39"
   #  - "40"
 spackver:
-  #- "0.20.3"
   - "0.21.0"
+  - "0.22.0"
```

Comment:
Let's use `0.21.3` and `0.22.3` to be at the latest patch version.

Reply:
Yes, and I will also include 0.23.0

Reply:
Done.